languages/nix.toml

```
@@ -0,0 +1,26 @@
+name = "nix"
+entrypoint = "main.sh"
+extensions = [
+  "sh"
+]
```

```suggestion
extensions = [
  "sh"
]
popularity = 5.0
```

that way nix will never be auto-suggested where bash should.